### PR TITLE
creation of the navbar with navigation in the different pages

### PR DESCRIPTION
**Pull request by sdiakhate**

### CODE REVIEW

File: frontend/src/App.jsx [hunk 1]
```diff
@@ -1,40 +1,21 @@
...
-            className="App-link"
-            href="https://vitejs.dev/guide/features.html"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            Vite Docs
-          </a>
-        </p>
-      </header>
+    <div className="container">
```
Comment (ioayoub) [inline frontend/src/App.jsx:11]: This is the old way to use React-router-dom, please remove it for now.

File: frontend/src/main.jsx [hunk 2]
```diff
@@ -1,21 +1,12 @@
...
-  },
-]);
-
-const root = ReactDOM.createRoot(document.getElementById("root"));
-
-root.render(
+ReactDOM.createRoot(document.getElementById("root")).render(
   <React.StrictMode>
-    <RouterProvider router={router} />
+    <BrowserRouter>
```
Comment (ioayoub) [inline frontend/src/main.jsx:8]: Same here, please remove this, old way to use react-router-dom

**Review by ioayoub — CHANGES_REQUESTED** [review]

File: frontend/src/App.jsx (hunk 1 above)
Comment (sdiakhate) [reply frontend/src/App.jsx:11]: ok I will modify

Comment (sdiakhate) [reply frontend/src/App.jsx:6]: I deleted the react-router-dom. You can double-check

**Review by ioayoub — APPROVED** [review]

**Pull request merged**